airbyte-integrations/connectors/source-outbrain-amplify/source_outbrain_amplify/spec.yaml

```diff
@@ -0,0 +1,85 @@
+documentationUrl: 'https://docsurl.com'
+connectionSpecification:
+  $schema: 'http://json-schema.org/draft-07/schema#'
+  title: Outbrain Amplify Spec
+  type: object
+  required: ["credentials", "start_date"]
+  properties:
+    credentials:
+      title: Authentication Method
+      description: >-
```

Comment:
Authentication methods that are secrets (ex: password, etc) should be marked with the airbyte_secret field